sympy/physics/mechanics/functions.py

```diff
@@ -81,7 +81,8 @@ def inertia(frame, ixx, iyy, izz, ixy=0, iyz=0, izx=0):
     (N.x|N.x) + 2*(N.y|N.y) + 3*(N.z|N.z)
 
     """
-
+    if isinstance(frame, Body):
```

Comment:
Body class is not defined in this file. Please add the following line at the start of this file with the other import statements.
```python
from sympy.physics.mechanics.body import Body
```